src/test/java/uk/org/tombolo/lac/LAQNImporterTest.java

```diff
@@ -62,7 +62,7 @@ private Object allowAccessToPrivateMethods(String methodName,
 
     @Test
     public void testGetProvider() throws Exception {
-        laqnImporter.importDatasource("airQualityControl",
+        laqnImporter.importDatasource("airQualitySensor",
```

Comment:
As mentioned in the first the comment on the top, now you can have "airQuality" here and in the other `importDatascource()` methods in the test.